src/structs/Transitions.jl

```diff
@@ -19,9 +36,45 @@ struct Rotation{T<:Real} <: AbstractTransition
     center::Union{Nothing,Point,AbstractObject}
 end
 
+"""
+    anim_rotate
+
+Animate the rotation of the attached object (see [`act!`](@ref)).
+Similiar function: [`anim_rotate_around`](@ref) to rotate around a point
+
+# Example
+```julia
+Background(1:100, ground)
+obj = Object((args...) -> rect(O, 50, 50, :fill), Point(100, 0))
+act!(obj, Action(1:50, anim_rotate(2π)))
+```
+
+# Options
+- `anim_rotate(ta::Real)` define the end angle of the rotation
+- `anim_rotate(fa::T, ta::T)` define the from and end angle
+
+"""
 anim_rotate(ta::Real) = Rotation(0.0, ta, nothing)
 anim_rotate(fa::T, ta::T) where {T<:Real} = Rotation(fa, ta, nothing)
 
+"""
+    anim_rotate_around
+
+Animate the rotation of the attached object (see [`act!`](@ref)) around a point.
+Similiar function: [`anim_rotate`](@ref) to rotate or spin an object
+
+# Example
+```julia
+Background(1:100, ground)
+obj = Object((args...) -> rect(O, 50, 50, :fill), Point(100, 0))
+act!(obj, Action(1:50, anim_rotate_around(2π)))
```

Comment:
Is this option defined in the Options section here? I don't clearly see it.

Reply:
Which option?

Reply:
I forgot defining a position there. Copy and paste error :smile:  
Thanks